packages/scripts/config/webpack.config.js

```diff
@@ -76,6 +76,12 @@ const getJsonpFunctionIdentifier = () => {
 	);
 };
 
+const getLiveReloadPort = ( inputPort, defaultPort = 35729 ) => {
+	const parsedPort = parseInt( inputPort, 10 );
+
+	return Number.isInteger( parsedPort ) ? parsedPort : defaultPort;
```

Comment:
This function is used only in one place so the default port set as a function param is not needed.
```suggestion
	return Number.isInteger( parsedPort ) ? parsedPort : 35729;
```

Reply:
Applied suggestion